opentelemetry-sdk/src/metrics/pipeline.rs

```diff
@@ -385,7 +386,11 @@ where
                 .clone()
                 .map(|allowed| Arc::new(move |kv: &KeyValue| allowed.contains(&kv.key)) as Arc<_>);
 
-            let b = AggregateBuilder::new(self.pipeline.reader.temporality(kind), filter);
+            let b = AggregateBuilder::new(
+                self.pipeline.reader.temporality(kind),
+                filter,
+                stream.cardinality_limit.unwrap_or(2000),
```

Comment:
I think we can make the code more readable / self-explanatory by adding a `const CARDINALITY_LIMIT_DEFAULT` in the file and use that instead of directly using `2000`.